datafusion/physical-expr/Cargo.toml

```diff
@@ -57,3 +57,4 @@ rand = "0.8"
 regex = { version = "^1.4.3", optional = true }
 sha2 = { version = "^0.10.1", optional = true }
 unicode-segmentation = { version = "^1.7.1", optional = true }
+uuid = { version = "^1.2", features = ["v4"] }
```

Comment:
Moar dependencies 😢

Reply:
unlike the:
- #3993 
one i'd argue that `uuid` is a bit ubiquitous so this should be fine